string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,8 @@
+/* exported capitalize */
+function capitalize(word) {
+  var small = word.toLowerCase();
+  var capitalOne = small.slice(0, 1).toUpperCase();
```

Comment:
You should do `small[0].toUpperCase()` instead, you don't need to use `slice` to view a character at an index.